src/main/java/frc/robot/config/ElevatorConfig.java

```diff
@@ -11,6 +11,7 @@ public class ElevatorConfig {
   public static final double EL_G = 0.0;
   public static final double EL_V = 0.0;
   public static final double EL_A = 0.0;
+  public static final double EL_PERIOD = 0.2;
```

Comment:
Instead of creating our own constant, use the `TimedRobot.kDefaultPeriod` constant